### DISCUSSION TITLE
Full Page QSplitter extending past screen

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#12768] [General - Components / Directives / etc]
I'm trying to make a full page splitter however there are two issues I'm not sure how to resolve..

https://codepen.io/jokermartini/pen/YzYzGQL

1. If you drag the splitter to the bottom of the page, it for some odd reason extends past the screen and shows double scrollbars on the side then.
2. I've used the slot:splitter but the dragging of the splitter doesn't stop so the splitter component is always visible on screen. How do i resolve that? Do i need to the set the property `separator-style="height: 6px"` in order for that to work?

![image](https://user-images.githubusercontent.com/1416749/158023259-212faf14-6a3d-4c05-bdb8-ceb1cd1f0a54.png)

**Comment by JokerMartini** [comment]
I want to it look something like this. But for some reason when overriding the separator, the height is not taken into account. This seems like a bug to me.

![image](https://user-images.githubusercontent.com/1416749/158023376-847a09de-6805-4e84-8374-f0b7195db12d.png)

**Comment by JokerMartini** [comment]
Any advice on how to solve this?

**Comment by pdanpdan** [comment]
Either use an overflow hidden in the splitter, or set smaller upper limit for the split, or add some space below the splitter, ...
